config.go

```diff
@@ -32,6 +32,7 @@ type agentPeerConfig struct {
 // servers.
 type agentDeviceConfig struct {
 	Name  string            `json:"name"`
+	MTU   int               `json:"mtu"`
```

Comment:
Should we default to `device.DefaultMTU` in `verifyAgentOidcConfig` ?

Reply:
I do not have a strong preference here. It felt better to allow default (0) and change to device.DefaultMTU in tun.go, so that the config doesn't know anything about tun

Reply:
It's just that I don't see this behaviour documented anywhere (setting to 0 means "default") and digging deeper into the `tun` code it seems like it just passes the value all the way to the `SIOCSIFMTU` syscall, for which I can't find any reference to the behaviour of setting to zero.

Reply:
https://github.com/utilitywarehouse/wiresteward/pull/34/files#diff-0b1a52e57fe9a28d5c2cac66892872d5R26-R29 I am doing what you suggested in tun.go, I do not think we are in the same page. I thought you are just asking for this to move to config.go

Reply:
<include facepalm>
Really sorry Foivos!

Reply:
np! :))